script/compile

```diff
@@ -59,13 +66,21 @@ args=( "-jar" "$JAR"
        "-H:Log=registerResource:"
        "-H:EnableURLProtocols=http,https"
        "--enable-all-security-services"
+       "--allow-incomplete-classpath"
```

Comment:
Without this flag, the compilation would fail with a recommendation for adding this flag.